force-app/main/default/classes/ClassesObjectsMethods.cls

```diff
@@ -40,17 +40,22 @@ public with sharing class ClassesObjectsMethods {
      */
     public static List<String> practiceStringMethods() {
         // Initialize a String with 'Hello, Salesforce!'
-        
-        // Get the length of the String        
+        String str1 = 'Hello, Salesforce!';
+        // Get the length of the String 
+        Integer stringLength = str1.length();       
         // Get a substring of the String of the first 5 characters
+        string firstFiveChar = str1.substring(0, 5);
         // Convert the String to lower case
-        
+        String lowerCaseString = str1.toLowerCase();
         // Return the list with the string data
-        // In the first position add the string length
+       List<String> stringData = new List<String>();
+       // In the first position add the string length
+        stringData.add(String.valueOf(stringLength));
         // In the second position add the substring
+        stringData.add(firstFiveChar);
         // In the third position add the string in lower case
-
-        return null; // Replace null with the variable you used to store the result
+        stringData.add(lowerCaseString);
+        return stringData; // Replace null with the variable you used to store the result
```

Comment:
Hi Bruce your logic is fine. From a style point of view new devs tend to create a lot of variables that are not needed. I think that is fine if helps with reliability. But in this case, I think you can do something like below and not lose anything.
```
  public static List<String> practiceStringMethods() {
        List<String> strings = new List<String>();
        String helloSalesforce = 'Hello, Salesforce!';
        strings.add(String.valueOf(helloSalesforce.length()));
        strings.add(helloSalesforce.substring(0, 5));
        strings.add(helloSalesforce.toLowerCase());
        return strings;
    }
    ```